grails-app/views/returns/show.gsp

```diff
@@ -130,7 +130,14 @@
                                 <g:message code="stockMovement.status.label"/>
                             </td>
                             <td class="value">
-                                <format:metadata obj="${stockMovement?.status ?: (stockMovement?.shipment?.status?.code ?: stockMovement?.statusCode)}"/>
+                                <g:set var="isInbound" value="${stockMovement?.destination == currentLocation}" />
+                                %{-- If it's an inbound return, we want to display a shipment status--}%
```

Comment:
I feel like this should not be here because we wanted to move away from this type of logic in templates. Hopefully, this won't have to be drastically changed, after getting the results of the spike that @drodzewicz is working on.